### PR TITLE
Feature/sync permissions command

### LINKED ISSUE #406: Discover and implement a way to seed and manage permissions properly
To implement this, we need to:

- [x] Decide on a naming convention for permissions (i.e. "edit article", "edit-article" or "article-edit")
- [x] Design an xml/json/yaml/csv-like format that defines all the roles and permissions for the app, which can be version controlled by Git OR decide if we are creating a roles and permissions config php file in the `/config` folder
- [x] Implement an artisan command (i.e. `php artisan permissions:upsert`) that synchronizes the roles and permissions in the database with the definition files

**Edit:**
We need to change the term 'upsert' into 'synchronize'. This because upsert will either insert new or update exiting items and does NOT delete items that shouldn't be in the database anymore.

### PR DESCRIPTION
**Pull request by dwaard**
# Description

closes #406 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## What needs to be tested

- [ ] Check if different users can access their views
- [ ] check if different users can create/update/delete their things
- [ ] Review the wiki page [Users & Policies](https://github.com/HZ-HBO-ICT/it-conference/wiki/Users-&-Policies) to check if the users, roles and permissions are aligned with the permissions configuration YML file
- [ ] Review the wiki page [Permission management](https://github.com/HZ-HBO-ICT/it-conference/wiki/Permission-management) to check if the content is clear for other developers to use the permissions and update the config file

# Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: resources/views/myhub/company/details.blade.php [hunk 1]
```diff
@@ -25,7 +25,7 @@
                         @livewire('company.manage-logo', ['company' => $company])
```
Comment (v-stamenova) [inline resources/views/myhub/company/details.blade.php:25]: When I am a company representative and the company is approved I am unable to edit the company logo
image

File: resources/views/myhub/company/details.blade.php [hunk 2]
```diff
@@ -25,7 +25,7 @@
...
                         <x-section-border/>
                         @livewire('company.details-form', ['company' => $company ])
```
Comment (v-stamenova) [inline resources/views/myhub/company/details.blade.php:27]: The same case with the editing of company details

image

File: resources/views/myhub/company/details.blade.php [hunk 3]
```diff
@@ -25,7 +25,7 @@
...
-                        @can('viewMembers', $company)
+                        @can('viewAnyMember', $company)
                         @livewire('company.member-manager', ['company' => $company])
```
Comment (v-stamenova) [inline resources/views/myhub/company/details.blade.php:29]: When member is invited and try to register via the email this error is thrown:
image

Edit: Saw the opened issue

**Review by v-stamenova — CHANGES_REQUESTED** [review]
Considering the open issue, after fixing the editing of company details I think it's safe to merge

File: resources/views/myhub/company/details.blade.php (hunk 2 above)
Comment (dwaard) [reply resources/views/myhub/company/details.blade.php:27]: Fixed in [21cc775](https://github.com/HZ-HBO-ICT/it-conference/pull/411/commits/21cc7752c176550b619e7452da865ba6368d5446)

File: resources/views/myhub/company/details.blade.php (hunk 1 above)
Comment (dwaard) [reply resources/views/myhub/company/details.blade.php:25]: Fixed in [21cc775](https://github.com/HZ-HBO-ICT/it-conference/pull/411/commits/21cc7752c176550b619e7452da865ba6368d5446)

File: resources/views/myhub/company/details.blade.php (hunk 3 above)
Comment (dwaard) [reply resources/views/myhub/company/details.blade.php:29]: Wontfix then

**Pull request merged**